parsons/utilities/dbt/__init__.py

```diff
@@ -0,0 +1,60 @@
+"""Utility for running and logging output from dbt commands
+
```

Comment:
These docs are good, but ideally we want to be able to surface them on the docs website. Maybe add a hook into the autodocs on this page? https://move-coop.github.io/parsons/html/stable/utilities.html#

Reply:
I thiiink 8d56b8a5da5172597ab6b162cf34d0ec0f63a473 should do this